src/Management/test/Endpoint.Test/Actuators/Health/EndpointMiddlewareTest.cs

```diff
@@ -361,19 +363,18 @@ public async Task GetStatusCode_MicrosoftAggregator_ReturnsExpected()
 
         builder.ConfigureAppConfiguration((_, configuration) => configuration.AddInMemoryCollection(new Dictionary<string, string?>(_appSettings)
         {
-            ["HealthCheckType"] = "default"
+            { "HealthCheckType", "default" }
         }));
 
         using IWebHost app = builder.Build();
         await app.StartAsync();
 
         using HttpClient client = app.GetTestClient();
         HttpResponseMessage unknownResult = await client.GetAsync(new Uri("http://localhost/cloudfoundryapplication/health"));
-        Assert.Equal(HttpStatusCode.OK, unknownResult.StatusCode);
+        unknownResult.StatusCode.Should().Be(HttpStatusCode.OK);
```

Comment:
Why is this called `unknownResult`? It implies the status is `Unknown`, which isn't actually the case. Should just be named `response`.

Reply:
That's a question I pondered for a few minutes yesterday, but digging below the surface just got me more confused... Like how is the "default" HealthCheckType going to use the Microsoft Aggregator?? I didn't want to get more involved once I saw that and just left the rest for now